labgrid/remote/exporter.py

```diff
@@ -341,6 +341,26 @@ def _get_params(self):
             'control_path': self.local.control_path,
         }
 
+@attr.s(eq=False)
+class YKUSHPowerPortExport(USBGenericExport):
+    """ResourceExport for YEPKIT YKUSH switchable USB hub"""
+
+    def __attrs_post_init__(self):
+        super().__attrs_post_init__()
+
+    def _get_params(self):
+        """Helper function to return parameters"""
+        return {
+            'host': self.host,
+            'busnum': self.local.busnum,
+            'devnum': self.local.devnum,
+            'path': self.local.path,
+            'vendor_id': self.local.vendor_id,
+            'model_id': self.local.model_id,
```

Comment:
It seems, that most of these are not used by the driver. It would be better to tell ykushcmd the busnum/devnum or path, so it wouldn't need to scan. Maybe that's supported?